### DISCUSSION TITLE
Next Gen Images

### DISCUSSION THREAD

**Discussion by Tigerlady** [MasaCMS/MasaCMS#181] [Ideas]
Would like to see that we can start using the next Gen images.  Like SVG or WEBP.  When uploading an image for a page it would be great to have a few options to upload next gen so when a browser accepts them it is what is used.  Would anyone else find this helpful?

**Comment by pshaughnessy88** [comment]
I agree and would add AVIF files to the list.

**Comment by guustnieuwenhuis** [comment]
@Tigerlady @pshaughnessy88 could you create a feature request for this? Please describe what you expect, where you expect it and how you would use it.
